spec/support/capybara.rb

```diff
@@ -9,4 +10,6 @@
   Capybara.register_driver(:poltergeist) do |app|
     Capybara::Poltergeist::Driver.new app, js_errors: true, timeout: 60
   end
+
+  Capybara.save_and_open_page_path = ENV['CIRCLE_ARTIFACTS'] if ENV['CIRCLE_ARTIFACTS']
```

Comment:
@damianlegawiec Hi, just curious how come you add Circle CI related stuff to the repos when they run on Travis and also lack circle.yml? By the way, nice work you guys do keeping the spree repositories maintained :+1: